spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
 require "rails_helper"
 
-show_tests_in_browser = false
-do_not_show_tests_in_browser = false
+feature "Photos:" do
+
+  context "photos show page" do
```

Comment:
Below, it looks like the expectations are on `/photos` — should this read `context "index page"`? (`"Photos"` will already be prefixed from the `feature` above, no need to repeat?)

Bear in mind this guideline:

https://teachersmanual.firstdraft.com/how-to-write-tests.html#avoid-deeply-nesting-describefeature-context-itscenario